### PR TITLE
ESLint

### PR DESCRIPTION
**Pull request by SubJunk**

### CODE REVIEW

**Comment by SubJunk** [comment]
@js-kyle keen to know if the addition of this `.vscode/settings.json` file is enough to make your VS Code ESLint plugin work for the TypeScript files in here (if you still use that)

**Comment by js-kyle** [comment]
@SubJunk it worked after I ran `yarn` on this branch and restarted Code 👍

File: .travis.yml [hunk 1]
```diff
@@ -4,12 +4,16 @@ node_js:
   - "12"
 
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.19.1
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.21.1
```
Comment (js-kyle) [inline .travis.yml]: We could maybe get this from package.json dynamically?

File: .travis.yml [hunk 2]
```diff
@@ -4,12 +4,16 @@ node_js:
...
   - yarn run build
 
 cache:
   directories:
-    - $HOME/.yarn
\ No newline at end of file
+    - $HOME/.yarn
+
+script:
+  - node_modules/.bin/eslint src/**/*.ts
```
Comment (js-kyle) [inline .travis.yml]: Do we want to run it for test files too?

File: .travis.yml (hunk 1 above)
Comment (SubJunk) [reply .travis.yml]: Yup sounds good

File: .travis.yml (hunk 2 above)
Comment (SubJunk) [reply .travis.yml]: Yup sounds good

**Pull request merged**